### PR TITLE
Modified uploader json file for command /sbin/lspci -k

### PR DESCRIPTION
**Pull request by vishwanathjadhav**
Parser:  https://github.com/RedHatInsights/insights-core/pull/1252

Trello: https://trello.com/c/FMhHS3Nf

### CODE REVIEW

File: uploader.json [hunk 1]
```diff
@@ -387,6 +387,11 @@
             "pattern": [],
             "symbolic_name": "lspci"
         },
+        {
+            "command": "/sbin/lspci -k",
+            "pattern": [],
+            "symbolic_name": "lspci_kernel"
+        },
```
[the remark this answers is not in the document]
Comment (vishwanathjadhav) [reply uploader.json]: Done

**Comment by vishwanathjadhav** [comment]
@SteveHNH, @wcmitchell  Please merge this PR.

**Comment by bfahr** [comment]
@SteveHNH @wcmitchell This json cannot be merged until the insights-core PR has been merged, which will break some rules and therefore production.  So please hold off on merging this PR until we can coordinate the insights-core PR merge and rule updates.

**Comment by xiangce** [comment]
@vishwanathjadhav - I submitted a patch to support `first_of` function to the branch `lspci_k_command` of your forked repo. Please check it when you are back.

**Comment by xiangce** [comment]
@bfahr - makes sense to me, @vishwanathjadhav I closed the PR to your repo. And please rebase your branch and remove the old `lspci_kernel`

**Comment by xiangce** [comment]
Looks good to me!

File: uploader.json [hunk 2]
```diff
@@ -4373,15 +4368,6 @@
...
-        "lspci": {
-            "host": [
-                {
-                    "archive_file_name": "/insights_commands/lspci",
-                    "command": "/sbin/lspci",
-                    "pattern": []
-                }
-            ]
-        },
         "lssap": {
```
Comment (bfahr) [inline uploader.json]: @vishwanathjadhav please remove this change from your PR, this section of the json should not be changed.

Comment (vishwanathjadhav) [reply uploader.json]: Done please check

**Comment by YakovLittle** [comment]
test this please

**Comment by esakaiev** [comment]
test this please

**Comment by bfahr** [comment]
@SteveHNH @wcmitchell This PR is approved and needs to go in with this insights-core PR, https://github.com/RedHatInsights/insights-core/pull/1283

**Pull request merged**